src/internet_identity/src/storage/anchor.rs

```diff
@@ -369,8 +397,8 @@ fn check_anchor_invariants(devices: &Vec<&Device>) -> Result<(), AnchorError> {
 
     /// Single devices can use >500 bytes for the variable length fields alone.
     /// In order to not give away all the anchor space to the device vector, we limit the sum of the
```

Comment:
"In order to not give away all the anchor space to the device vector **and the identity metadata**"?
Or reword the whole comment.